### DISCUSSION TITLE
Radio Button with other choice enable

### DISCUSSION THREAD

**Discussion by Puspa-Subedi** [harness-software/wp-graphql-gravity-forms#417] [Q&A]
I am trying to implement the form from Wordpress to next js and while submitting the form data I am unable to submit the data if I select other and can't find the example to submit value from the input field that is displayed after other checked. I am a beginner with this so I am helpful if you provide me the example like for checkboxes and other.

**Comment by justlevine** [comment]
Hey @Puspa-Subedi 

As noted in the below discussion, the syntax for radio fields is `{id: $id, value: "My Custom Choice Value"}`, but there was conflict in recent GF core versions that prevented this from working. 

A patch is in the works at #421 , and will be shipped in the upcoming v0.13.0 release.

https://github.com/AxeWP/wp-graphql-gravity-forms/discussions/390#discussioncomment-9663561
